instrumentation-docs/src/test/java/io/opentelemetry/instrumentation/docs/utils/YamlHelperTest.java

```diff
@@ -134,7 +136,7 @@ void testGenerateInstrumentationYamlSeparatesClassifications() throws Exception
 
     InstrumentationMetaData internalMetadata =
         new InstrumentationMetaData(
-            null, InstrumentationClassification.INTERNAL.toString(), null, null);
+            null, InstrumentationClassification.INTERNAL.toString(), null, null, null);
```

Comment:
[minor] in a follow-up PR, switching to a builder pattern might help to make it more readable as there is now 5 parameters.

Reply:
i agree, created #14623 to track that work in a followup